behaviors/clubMatches.py

```diff
@@ -1,13 +1,18 @@
 import discord
 from discord.ext import commands
 from discord_slash import cog_ext, SlashContext
-import random
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+guilds = int(os.getenv('GUILD_ID'))
```

Comment:
Consider abstracting this away such that we dont have to repeat these same 4 lines for every cog.

Reply:
Not a bad Idea. I'll add that a bit later.